### PR TITLE
fix banner

### PR DESCRIPTION
**Pull request by dmitrig01**
@jsaleigh 

looks like the minifier shat on our variable names

### CODE REVIEW

**Comment by chanian** [comment]
:+1:

File: src/5_banner.js [hunk 1]
```diff
@@ -125,13 +125,10 @@ banner = function(branch, options, linkData, storage) {
...
-				if (err) {
-					throw err;
-				}
-				doc.getElementById('branch-mobile-action').onclick =
-					'branch.deepviewCta()';
-			});
+			branch['deepview'](linkData, options);
+			doc.getElementById('branch-mobile-action').onclick = function() {
+				branch['deepviewCta']();
+			};
```
Comment (jsaleigh) [inline src/5_banner.js]: could this create a race condition? it seems possible (though unlikely) that the call to branch['deepview'] could still be waiting for the ajax request to finish, when the element is clicked on

Comment (dmitrig01) [inline src/5_banner.js]: deepviewCta puts it on the queue, which will wait for deepview to complete

**Comment by jsaleigh** [comment]
:+1:

**Pull request merged**